src/index.js

```diff
@@ -23,6 +23,7 @@ export {default as InteractiveMap} from './components/interactive-map';
 export {default as StaticMap} from './components/static-map';
 export {default as MapControls} from './components/map-controls';
 export {default as Marker} from './components/marker';
+export {default as NavigationControl} from './components/navigation-control';
```

Comment:
I'd put these additional components in their own section below, before the overlays. I guess I see them as "extras".